config/config_test.go

```diff
@@ -33,6 +34,10 @@ endpoints:
       url: https://rpc.ankr.com/eth
       http_headers:
         user-agent: rss3-node
+    mastodon:
+      url: https://30.10.000.00:9092/
```

Comment:
```suggestion
The endpoint should be defined in the configuration file rather than hardcoding them. Sensitive information like this should not appear in the code, as it poses a security risk.
```

Reply:
The url is a mock url that is particularly used for this config_test file. Changed its value to https://0.0.0.0:9092/ for better view.